From 49f3aebcc8614d483c5753109a9d65aa33d301ea Mon Sep 17 00:00:00 2001
From: Jakub Hrozek <jhrozek@redhat.com>
Date: Tue, 11 Jun 2013 12:48:06 +0200
Subject: [PATCH 05/12] Use the correct talloc context when creating AD
 subdomains

sdom was only ever guaranteed to be set when a new domain was being
created. sditer is a valid pointer in both cases, so just use that.
---
 src/providers/ad/ad_subdomains.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/providers/ad/ad_subdomains.c b/src/providers/ad/ad_subdomains.c
index f4eec6a48019d55436631487a6108be405254766..07b523df5466319739e1f44164b7f08156ea214b 100644
--- a/src/providers/ad/ad_subdomains.c
+++ b/src/providers/ad/ad_subdomains.c
@@ -120,7 +120,7 @@ ads_store_sdap_subdom(struct ad_subdomains_ctx *ctx,
         }
 
         /* Convert the domain name into search base */
-        ret = domain_to_basedn(sdom, sditer->dom->name, &basedn);
+        ret = domain_to_basedn(sditer, sditer->dom->name, &basedn);
         if (ret != EOK) {
             DEBUG(SSSDBG_OP_FAILURE,
                 ("Cannot convert domain name [%s] to base DN [%d]: %s\n",
-- 
1.8.2.1

